### PR TITLE
FPU JIT fixes, 1 GB Z3 RAM barrier fix, a few other trivial changes

### PR DESCRIPTION
**Pull request by FrodeSolheim**

### CODE REVIEW

File: jit/codegen_x86.cpp [hunk 1]
```diff
@@ -4733,9 +4733,16 @@ LOWFUNC(NONE,NONE,2,raw_ftan_rr,(FW d, FR s))
 }
 LENDFUNC(NONE,NONE,2,raw_ftan_rr,(FW d, FR s))
 
+#ifdef CPU_x86_64
+#define REX64 emit_byte(0x48);
```
Comment (rofl0r) [inline jit/codegen_x86.cpp]: the omission of the semicolon when calling the macro causes problems with 3rd party tools such as source code formatters, IDEs etc that do not implement a full C preprocessor.

suggested improval:
```c
#define REX64() emit_byte(0x48)
#else 
#define REX64()
```
and later
```REX64();```

Comment (FrodeSolheim) [inline jit/codegen_x86.cpp]: @rofl0r It if fixes an actual problem for you, you are welcome to create a pull request with a commit.

Comment (rofl0r) [inline jit/codegen_x86.cpp]: done https://github.com/tonioni/WinUAE/pull/90

**Comment by timofonic** [comment]
Any news about us or got merged or not? What happened?

**Pull request merged**